infrastructure/stacks/uec-poweruser-role/uec-pu-iam-services.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+          "Sid": "VisualEditor0",
```

Comment:
Can we have a more meaningful Sid if poss please?  PowerUserBasePolicy maybe?

Reply:
done